app/src/main/java/com/gh4a/adapter/timeline/DiffViewHolder.java

```diff
@@ -27,6 +30,9 @@
 
 class DiffViewHolder extends TimelineItemAdapter.TimelineItemViewHolder<TimelineItem.Diff>
         implements View.OnClickListener {
+    private static final float[] DIFF_SIZE_MULTIPLIERS = new float[] {
+            0.5F, 0.75F, 1F, 1.5F, 2F
```

Comment:
Maybe use values in `sp` instead? Will be easier to setup exact values. Right now smallest is almost unreadable while in code viewer it looks ok.

Reply:
As those are only multipliers over the (style-set) default text size, I think the size is set in sp already (and the scaling for sp is global anyway). If we want to keep that setting being used here, I think we'll need to adapt the default text size to match 'normal' code viewer size and adapt the multipliers to make the other sizes match.+